cmd/rook/block/unmap.go

```diff
@@ -29,31 +29,31 @@ import (
 )
 
 var (
-	unmountDeviceName string
-	unmountPath       string
+	unmapDeviceName string
+	unmapPath       string
 )
 
-var unmountCmd = &cobra.Command{
-	Use:   "unmount",
-	Short: "Unmounts and destroys a block device from the local machine",
+var unmapCmd = &cobra.Command{
+	Use:   "unmap",
+	Short: "Unmap and destroys a block device from the local machine",
 }
 
 func init() {
-	unmountCmd.Flags().StringVar(&unmountDeviceName, "device", "", "Name of device to unmount (e.g., rbd0)")
-	unmountCmd.Flags().StringVar(&unmountPath, "path", "", "File system path to unmount")
+	unmapCmd.Flags().StringVar(&unmapDeviceName, "device", "", "Name of device to unmap (e.g., rbd0)")
+	unmapCmd.Flags().StringVar(&unmapPath, "mount", "", "File system mount point to unmount before unmaping")
```

Comment:
Yes, as new functionality it can be a new issue.  But I think it needs to be added, because unmap can also be seen as unmapping a Ceph block image.  Much as `unmount` specifies either the block device or the mount point, unmap is one level lower and should specify either the Ceph image or the block device.